tests/director/external-services-mock-integration/auditlog_integration_test.go

```diff
@@ -0,0 +1,67 @@
+package external_services_mock_integration
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/kyma-incubator/compass/tests/director/pkg/gql"
+	"github.com/kyma-incubator/compass/tests/director/pkg/idtokenprovider"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/tests/director/pkg/ptr"
+)
+
+func TestAuditlogIntegration(t *testing.T) {
+	ctx := context.Background()
+	httpClient := http.Client{}
+	appName := "app-for-testing-auditlog-mock"
+	appInput := graphql.ApplicationRegisterInput{
+		Name:         appName,
+		ProviderName: ptr.String("compass"),
+	}
+
+	t.Log("Get Dex id_token")
+	dexToken, err := idtokenprovider.GetDexToken()
+	require.NoError(t, err)
+
+	dexGraphQLClient := gql.NewAuthorizedGraphQLClient(dexToken)
+
+	t.Log("Register Application through Gateway with Dex id token")
+	registerApplicationFromInputWithinTenant(t, ctx, dexGraphQLClient, testConfig.DefaultTenant, appInput)
```

Comment:
I propose to create the request here, then pass it to client to execute and then we can compare it with version from auditlog logs